charts/generic/k8-custom-metrics-apiserver/k8-custom-metrics-apiserver.chartserviceversion.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: litmuchaos.io/v1alpha1
```

Comment:
@dharmaanu , this is a custom solution and don't have kubernetes resource associate, we don't need this chaos-chart.